proxy/migrations/tools/verification.ts

```diff
@@ -12,7 +12,7 @@ export async function verify(contractName: string, contractAddress: string, cons
                 });
                 break;
             } catch (e: any) {
-                if (e.toString().includes("Already Verified")) {
+                if (e.toString().includes("Contract source code already verified")) {
```

Comment:
Let's change it in the next PR to develop like `e.toString().toLoweCase().includes("Already Verified")`